aspnetcore/security/authorization/resourcebased.md

```diff
@@ -36,7 +36,9 @@ Task<AuthorizationResult> AuthorizeAsync(ClaimsPrincipal user,
                           string policyName);
 ```
 
-# [ASP.NET Core 1.x](#tab/aspnetcore1x)
+::: moniker-end
+
+::: moniker range="<= aspnetcore-1.1"
```

Comment:
I guess we didn't standardize. I've been going with ...

::: moniker range=">= aspnetcore-2.0"
::: moniker range="< aspnetcore-2.0"

... on tab replacements. It hardly matters. These all go away soon enough. :smile: